src/serverOperations/booksServerOperations.js

```diff
@@ -0,0 +1,75 @@
+import { bases } from '../enums';
+import {
+    createRecord,
+    deleteRecord,
+    getRecords,
+    updateRecord,
+} from './crudOperations';
+import { fetchFromBase } from './fetchFromBase';
+
+export function getBookServerOperations(sessionID) {
+    return {
+        createBook: async (title, author, amount) => {
+            return await createRecord(
+                bases.BOOKS,
+                { title, author, amount },
+                sessionID,
+            );
+        },
+
+        /**
+         * @param params {{ title?:string, author?:string, amount?:number }}
+         * @param fromRecordID {number} -1 means that start from the least id
+         * @param recordAmount {number} -1 means maximum
+         * @return {Promise<{
+         * id:number,
+         * title:string
+         * author:string
+         * amount:number} |
+         * boolean |
+         * {
+         *  staus:number
+         *  message:string}>}
+         * */
+        getBooks: async (params, fromRecordID = -1, recordAmount = -1) => {
```

Comment:
Please get rid of magic values. Being described in JSDoc doesn't "unmagic" them. It should be a constant (if it really has to exist)

Reply:
Same for other places in code